quiche/src/packet.rs

```diff
@@ -789,8 +787,9 @@ pub fn verify_retry_integrity(
 
 fn compute_retry_integrity_tag(
     b: &octets::OctetsMut, odcid: &[u8], version: u32,
-) -> Result<aead::Tag> {
+) -> Result<Vec<u8>> {
     const KEY_LEN: usize = RETRY_AEAD_ALG.key_len();
+    const TAG_LEN: usize = RETRY_AEAD_ALG.key_len();
```

Comment:
Should this be ` RETRY_AEAD_ALG.tag_len();`?

Reply:
Ooops, yeah, fixed now. It just so happen that tag and key len are the same for AES128 GCM...